fv3core/fv3core/stencils/updatedzd.py

```diff
@@ -2,9 +2,14 @@
 from gt4py.gtscript import BACKWARD, FORWARD, PARALLEL, computation, interval
 
 import pace.dsl.gt4py_utils as utils
+
+# import pace.util
```

Comment:
I removed the import, but keeping the function calls because that's the convention we currently have in pace (see `fxadv.py`). Basically we had to change how selective validation is done, and we settled on commented the code out so that we can still run it manually when we need.